tools/grunt-tasks/minifyNames.js

```diff
@@ -6,15 +6,15 @@ var MAX_IMPORT_LENGTH = 140;
 var IMPORT_INDENT_PREFIX = "    ";
 
 var AutoGeneratedConstFile = 
-    "// Copyright (c) Microsoft Corporation. All rights reserved.\n" +
-    "// Licensed under the MIT License.\n" +
-    "// @skip-file-minify\n\n" +
-    "// ##############################################################\n" +
-    "// AUTO GENERATED FILE: This file is Auto Generated during build.\n" +
-    "// ##############################################################\n\n" +
-    "// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n" +
-    "// Note: DON'T Export these const from the package as we are still targeting ES3 this will export a mutable variables that someone could change!!!\n" +
-    "// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n\n";
+    "// Copyright (c) Microsoft Corporation. All rights reserved.\r\n" +
```

Comment:
This fixes the issues where once building you end up with files that "look" like they have changed.

It turns out that the issue was related to the different new line (\r\n) that the files are defaulted to and that this script was only creating the files with "\n"